install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -193,6 +193,19 @@ rules:
       - list
       - watch
 
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
```

Comment:
why do you need all this verbs?

Reply:
machine-api-operator needs to read if the metal3-mariadb-password already exists and then creates it if doesn't. So, we would need create, update, delete and get. And, I looked at the verbs used for other "secret" resources in the file for guidance.

Reply:
mm I'm confused, based on your comment and looking at the PR it seems you'd only need `get/create`

Reply:
Update verbs associated with secret in Role to just get and create.